sdk/search/azure-search-documents/src/main/java/com/azure/search/annotation/SimpleFieldProperty.java

```diff
@@ -0,0 +1,21 @@
+package com.azure.search.annotation;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+@Target({ElementType.FIELD})
+@Retention(RetentionPolicy.RUNTIME)
+public @interface SimpleFieldProperty {
+    boolean isKey();
+    boolean isRetrievable();
+    boolean isFacetable();
+    boolean isSearchable();
+    boolean isSortable();
+    boolean isFilterable();
+    String analyzer();
```

Comment:
Some of these can only be set on `SearchableField` and shouldn't appear here. Make sure they don't in the convenience classes as well.